src/3D/setup3d.F90

```diff
@@ -906,16 +919,20 @@ subroutine setup(fileName, fileType)
         do i = 1, nUnique
             if (fullTopoType(i) == topoEdge .and. fullBCType(1, i) == BCDefault) then
                 print *, 'There was a missing boundary condition for edge node:', uniquePts(:, i)
```

Comment:
Can you add `ERROR:` here (same below) to make this consistent with printed errors elsewhere.